x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/spatial/20_cartesian_centroid.yml

```diff
@@ -114,9 +118,6 @@ setup:
 
 ---
 "Test cartesian_centroid aggregation on cartesian shape shapes":
-  - skip:
```

Comment:
The test : `Test cartesian_centroid aggregation on cartesian shape shapes with grouping` is muted too, should we unmute it too?

Reply:
Good catch! Pushed a commit for that